server/sync/tools.py

```
@@ -59,6 +61,122 @@
+            show_id = item.ids.slug
+
+        if show_id is None:
+            # No resolvable ID — send the show item directly
+            request = TraktHistoryRequest(shows=[item])
+            result = await client_method(request)
+            _aggregate_summary(combined, result, operation)
+            continue
+
+        try:
+            season_ids = await _get_show_season_ids(show_id)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)